geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -550,9 +550,7 @@
 
   private final PersistentMemberManager persistentMemberManager;
 
-  private ClientMetadataService clientMetadataService = null;
-
-  private final Object clientMetaDatServiceLock = new Object();
+  private ClientMetadataService clientMetadataService;
```

Comment:
Looks like we're setting this field in the constructor -- let's make it `final`.

Reply:
👍